File: fibonacci.go
///usr/bin/true; exec /usr/bin/env go run "$0" "$@" ; exit
package main

import (
	"fmt"
	"time"
)

func Fib(n int) int {
	if n <= 2 {
		return n
	}
	return Fib(n-1) + Fib(n-2)
}

func main() {
	start := time.Now()
	Fib(40)
	secs := float64(time.Now().UnixNano()-start.UnixNano()) * 0.000000001
	fmt.Printf("golang... %f seconds\n", secs)
}

File: fibonacci.js
#!/usr/bin/env node

function fib(n) {
	if (n <= 2) return 1;
	return fib(n-1) + fib(n-2);
}

const start = Date.now();
fib(40);
const millis = Date.now() - start;
console.log(`node..... ${(millis / 1000).toFixed(6)} seconds`);
